README.md

```diff
@@ -228,6 +330,10 @@ var re = /na/gi;
 (* ml *)
 let obj = [%obj { property = "value" }]
 ```
+```reason
+/* re */
+let obj = [%obj {property: "value"}];
```

Comment:
Might be nice to show the reason sugar `{ "property": "value }` as an alternative here